repositories/candidate_records.go

```diff
@@ -398,3 +397,56 @@ func (r *Repo) loadCandidateRecord(result *models.CandidateRecord) error {
 
 	return nil
 }
+
+func (r *Repo) UpdateCandidateRecordEmbargoes() (int, error) {
+	var n int
+
+	psql := sq.StatementBuilder.PlaceholderFormat(sq.Dollar)
+	query := psql.Select("*", "-1 AS total").
```

Comment:
Don't like this `-1 AS total` but it's kinda necessary to reuse the `queryRows` function below. Tried many ways but couldn't find a way to make this logic more generic.

Reply:
pretty it ain't, but I get why...